src/viam/sdk/tests/mocks/mock_robot.hpp

```diff
@@ -17,6 +17,9 @@ using namespace viam::sdk;
 // we should update the tests here accordingly.
 //
 class MockRobotService : public RobotService_ {
+   public:
+    MockRobotService(std::shared_ptr<Server> server) : RobotService_(server){};
```

Comment:
Needed this constructor for `robot_client_to_mocks_pipeline` and it had to be public.